src/server/services/Analytics.js

```diff
@@ -17,30 +28,37 @@ async function getAnalytics() {
   const dbAnalytics = client.db(dbName);
   const cursor = await dbAnalytics.collection('analytics').find();
   const result = await cursor.toArray();
-
+  result.forEach(function(x) {
```

Comment:
@finnzink A cleaner way to do this would to tell MongoDB to not return _id. See https://stackoverflow.com/a/22121172/6001

Reply:
@rlorenzo Those examples are for a single returned object - it doesn't seem to work when you call .find() without any parameters. Open to suggestions

Reply:
@finnzink You tried the following?

`const cursor = await dbAnalytics.collection('analytics').find({},{_id: false});`

It didn't work?

Reply:
Unfortunately not. _id field still shows up.

Reply:
@finnzink Strange.. okay, let's leave it alone